### PR TITLE
Kan 6 harsha payment

### PR DESCRIPTION
**Pull request by Harsha1varthan**

### CODE REVIEW

**Comment by aravind-digitech** [comment]
@Harsha1varthan, pls add a short description similar to other PRs

File: Frontend/src/App.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+.CheckOut{
+    display: grid;
+    background-color: var(--bg-white);
+    align-items: center;
+    min-height: 100vh;
+}
+.CheckOut_Header{
+    padding: 1rem;
+    box-shadow:  4px 6px rgba(15, 15, 15, 0.08)
```
Comment (aravind-digitech) [inline Frontend/src/App.css:14]: use variables

File: Frontend/src/App.jsx
Comment (aravind-digitech) [inline Frontend/src/App.jsx:1]: pls revert changes in this file

File: Frontend/src/Payment/Checkout.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    let slotClass = address ? 'CheckOut_box' : 'CheckOut_box_Inactive'
+
+    function handleProceed(){
+        navigate("/payment")
+    }
+
+    return(
+        <div className = "CheckOut">
+            <div className="CheckOut_Header">
+                <h1>Checkout</h1>
```
Comment (aravind-digitech) [inline Frontend/src/Payment/Checkout.jsx:25]: use constants as in https://github.com/AYS-At-Your-Service/AYS/pull/13/files#diff-c2c392d453cc978fcfed88aae84cdeb1b2ce75f60d0bb32f26cc6e821ab39a7b

File: Frontend/src/Payment/Payment.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+export function Payment(){
+    return(
+        <div style={{width: "100%", minHeight : "100vh", display: "grid", alignItems: "center"}}>
+            <div className="UpiBox">
+                    <h2>UPI</h2>
+                    <div style={{display: "flex", width: "250px", justifyContent: "space-between", alignItems: "center"}}>
+                        <div style={{display: "flex", justifyContent: "center", alignItems: "center"}}>
+                            <img src = {upiImage} style = {{width: "60px", height: "60px", marginRight: "10px"}}/>
+                            <p>Pay via UPI Id</p>
```
Comment (aravind-digitech) [inline Frontend/src/Payment/Payment.jsx:13]: use constants

File: Frontend/src/assets/location.webp
Comment (aravind-digitech) [inline Frontend/src/assets/location.webp:1]: pls remove all bin files

**Review by aravind-digitech — CHANGES_REQUESTED** [review]

**Comment by aravind-digitech** [comment]
pls re raise PR once the changes are done

**Comment by Harsha1varthan** [comment]
Once It was done, I would raise the PR

**Pull request closed without merging**